includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -88,6 +88,13 @@ public function collect_style_elements() {
 				continue;
 			}
 
+			if ( 'body' === $style_element->parentNode->nodeName && $style_element->hasAttribute( 'amp-keyframes' ) ) {
+				$validity = $this->validate_amp_keyframe( $style_element );
```

Comment:
I guess this doesn't really have to be stored in a var and could part of the conditional statement above.

Reply:
The reason behind is is that in #912 we need to add reporting and `$validity` is a `WP_Error` instance, so we'd want to report it when this is merged into that PR.